avocado/utils/pci.py

```diff
@@ -135,7 +141,8 @@ def get_slot_from_sysfs(full_pci_address):
 
     :return: slot of PCI address from sysfs.
     """
-    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' % full_pci_address):
+    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' %
+                          full_pci_address):
```

Comment:
We do not enforce the 80 chars (even though we do our best to keep it), so it won't fail in the CI. Anyway if you want to keep your check happy you can reorganize the commits so first you fix the style fixes and then you add this commit.